model-optimizer/extensions/middle/L2NormToNorm.py

```diff
@@ -82,6 +82,27 @@ def replace_pattern(self, graph: Graph, match: dict):
             log.debug('The value of the "maximum_y_data" is not defined or is not constant')
             return
 
+        # We need to check axes which performed reduction because IE supports only 2D, 3D, 4D inputs and
```

Comment:
I suggest to completely refactor this transformation and generate operation `NormalizeL2`. Because currently we generate non-opset operation "Normalize" which is then converted to `NormalizeL2` in the `model-optimizer/extensions/back/NormalizeToNormalizeL2.py`. Moreover, the `NormalizeL2` op semantic is exactly what we need: it has `axes` input , and `eps_mode`. So there is no need to bother with `across_channels` and `across_spatial` at all :)

Reply:
Ok, done.